samza-core/src/main/java/org/apache/samza/config/JobCoordinatorConfig.java

```diff
@@ -23,7 +23,6 @@
 
 public class JobCoordinatorConfig extends MapConfig {
   public static final String JOB_COORDINATOR_FACTORY = "job-coordinator.factory";
```

Comment:
@navina Unrelated to this PR, but does this need to be changed to job.coordinator.factory?